src/internal/pachd/full.go

```diff
@@ -33,13 +33,15 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachconfig"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
 	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+	pjs_server "github.com/pachyderm/pachyderm/v2/src/internal/pjs"
```

Comment:
This stuff would ideally be in src/server/pjs.  That's just the convention in this repo.

Reply:
I think that it’s good to use an internal package for functionality that users of Pachyderm as a library will never touch.

Arguably this could also go in `src/server/pjs/internal/whatever`, if nothing outside of PJS should use it.

Reply:
For the record I'm not sure why this was put in this package. I can move it but I'll ask @Zhang-Muyang and/or @brendoncarroll first since there may have been a compelling reason.

Reply:
We talked about this on the WFC meeting 9/5/2024:

We want grpc servers to be in /internal/ since that means they can only be imported from sibling packages and not anything outside. Clients on the other hand should not be in internal since we want them to be useable everywhere.